front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx

```diff
@@ -30,6 +32,35 @@ import ImportTrainScheduleV2 from './ImportTrainScheduleV2';
 import ManageTrainScheduleV2 from './ManageTrainScheduleV2';
 import SimulationResultsV2 from './SimulationResultsV2';
 
+type MicroMacroSwitchProps = {
+  isMacro: boolean;
+  onClickButton: (isMacro: boolean) => void;
+  t: (key: string) => string;
+};
+
+const MicroMacroSwitch = ({ isMacro, onClickButton, t }: MicroMacroSwitchProps) => (
```

Comment:
I think this file is big enough, can you move this component in a new file ? You can create a new file operationalStudies/components/MicroMacroSwitch.tsx
We will move it in a proper folder later

Reply:
Done

Reply:
Thank you for your review